source/android/mobile/src/main/java/io/adaptivecards/adaptivecardssample/MainActivityAdaptiveCardsSample.java

```diff
@@ -398,50 +448,33 @@ private String loadFile(Uri uri)
         return null;
     }
 
-    private void loadAdaptiveCard(Intent data)
+    /**
+     * Load given AdaptiveCard
+     * @param fileName name of chosen card JSON file
+     * @param card file contents
+     */
+    private void loadAdaptiveCard(String fileName, String card)
     {
-
-        String fullString = loadFile(data.getData());
-        if (fullString.isEmpty())
+        if (card.isEmpty())
```

Comment:
Should we report some sort of a warning message via the tool if the card is empty since you are special handling this case?

Reply:
Note sure, this was the previous behavior. Should I add a toast here, @almedina-ms ?

Reply:
We weren't previously, but yeah we could show a toast. @almedina-ms shall I add that?